clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/ResourceExhaustionCalculatorTest.java

```diff
@@ -98,4 +100,45 @@ public void no_feed_block_returned_when_feed_block_disabled() {
         assertNull(feedBlock);
     }
 
+    @Test
+    public void retain_node_feed_block_status_when_within_hysteresis_window_limit_crossed_edge_case() {
+        var curFeedBlock = ClusterStateBundle.FeedBlock.blockedWith("foo", setOf(exhaustion(1, "memory", 0.51)));
+        var calc = new ResourceExhaustionCalculator(true, mapOf(usage("disk", 0.5), usage("memory", 0.5)), curFeedBlock, 0.1);
+        // Node 1 goes from 0.51 to 0.49, crossing the 0.5 threshold. Should still be blocked.
+        // Node 2 is at 0.49 but was not previously blocked and should not be blocked now either.
+        var cf = createFixtureWithReportedUsages(forNode(1, usage("disk", 0.3), usage("memory", 0.49)),
+                                                 forNode(2, usage("disk", 0.3), usage("memory", 0.49)));
+        var feedBlock = calc.inferContentClusterFeedBlockOrNull(cf.cluster().getNodeInfo());
+        assertNotNull(feedBlock);
+        // TODO should we not change the limits themselves? Explicit mention of hysteresis state?
+        assertEquals("memory on node 1 [storage.1.local] (0.490 > 0.400)",
+                     feedBlock.getDescription());
+    }
+
+    @Test
+    public void retain_node_feed_block_status_when_within_hysteresis_window_under_limit_edge_case() {
+        var curFeedBlock = ClusterStateBundle.FeedBlock.blockedWith("foo", setOf(exhaustion(1, "memory", 0.49)));
+        var calc = new ResourceExhaustionCalculator(true, mapOf(usage("disk", 0.5), usage("memory", 0.5)), curFeedBlock, 0.1);
+        // Node 1 goes from 0.49 to 0.48, NOT crossing the 0.5 threshold. Should still be blocked.
```

Comment:
Is it the adjusted threshold (0.4) we should use in the comment?

Reply:
In this case, the comment is referring to the original feed limit, not the adjusted hysteresis limit. It tests going from feed blocked above the hysteresis limit (but below the original limit) to feed blocked _slightly less_ above the hysteresis limit.